contract/test/builder/init-orca.js

```diff
@@ -0,0 +1,52 @@
+console.log('sane');
```

Comment:
If you're going with this sort of builder script, it's not just test material. It belongs in `contract/scripts/`.